internal/store/queuestore.go

```diff
@@ -15,49 +15,59 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-package target
+package store
 
 import (
 	"encoding/json"
+	"errors"
 	"os"
 	"path/filepath"
 	"sort"
 	"strings"
 	"sync"
 	"time"
 
-	"github.com/minio/minio/internal/event"
+	"github.com/google/uuid"
 )
 
 const (
 	defaultLimit = 100000 // Default store limit.
-	eventExt     = ".event"
+	defaultExt   = ".unknown"
```

Comment:
```suggestion
	defaultExt   = ".events"
```

Reply:
This is an extension used for the items saved in the store. Each and every item that uses the store have its own file extension. Bucket notification events use `.event` already. So, i kept `.unknown` as the default one.

Reply:
It is .event*s*  - but it shouldn't really be used.